orders/db/schema.cds

```diff
@@ -0,0 +1,17 @@
+using { sap.capire.bookshop.Books } from '@capire/bookshop';
+using { Currency, managed, cuid } from '@sap/cds/common';
+namespace sap.capire.bookshop;
+
+entity Orders : cuid, managed {
+  OrderNo  : String @title:'Order Number'; //> readable key
+  Items    : Composition of many OrderItems on Items.parent = $self;
+  total    : Decimal(9,2) @readonly;
```

Comment:
We don't have a good solution for the (dynamic) calculation of `Order.total`.
Shall we remove it for the time being?

Reply:
@wattnu FYI

Reply:
Yes, we should remove these fields for now

Reply:
See #34